libs/dh/message-archive/feature-log-search/src/lib/drawer/dh-message-archive-drawer.component.ts

```diff
@@ -113,9 +113,9 @@ export class DhMessageArchiveDrawerComponent implements OnInit, OnDestroy {
     this.message = null;
   }
 
-  getDocument(messageId: string) {
+  getDocument(id: string) {
     this.apiStore.getDocument({
-      id: messageId,
+      id: id,
```

Comment:
nitpick: when an object property matches with the name of a variable that we want to assign to it, a shorthand notation can be used.

```suggestion
      id,
```